workshop/1-Personalization/1.1-Personalize.ipynb

```diff
@@ -1668,7 +1668,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "product_id = 22\n",
+    "product_id = '020a5afe-fb13-4499-a1fa-8594d326eaa0'\n",
```

Comment:
FUTURE: I think we should change this to a randomly selected product from the items dataframe to make this code more future proof should the products.yaml file change again.